docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Delete and recover items
+---
+
+# Delete and recover items
+
+Sometimes your questions, dashboards, or models outlive their usefulness. You can send outdated items to **Trash**.
+
+![Move to trash](./images/move-to-trash.png)
+
+Items in **Trash** won't show up in search, and you won't be able to use them to create new questions and dashboards.
```

Comment:
You'll need to note here that you can search for items in the trash, if you need to.